Microsoft.MixedReality.WebRTC.sln

```diff
@@ -30,18 +30,24 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Test", "Test", "{35C3F3A6-2
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestNetCoreConsole", "examples\TestNetCoreConsole\TestNetCoreConsole.csproj", "{C17D2554-9409-4CC7-8337-E3FBE3CAE415}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestReceiveAV", "examples\TestReceiveAV\TestReceiveAV.csproj", "{209D1A4C-96F1-4F5E-9987-8C64E7998CC3}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU

```

Comment:
Remove this Solution config and all related Project configs below, they make no sense and make things very confusing for users regarding what architecture is being built for native projects. This was likely auto-added by VS, and we didn't pay attention.

Reply:
It seems it was added by the new sample project. Fixed.